behandlingslager/domene/src/main/java/no/nav/foreldrepenger/domene/typer/PersonIdent.java

```diff
@@ -107,6 +107,10 @@ public boolean erDnr() {
         return n > 3 && n <= 7;
     }
 
+    public boolean erFnr() {
+        return erGyldigFnr(ident);
```

Comment:
Mistenker at du får true for både FNR og DNR - "true hvis angitt str er et fødselsnummer (F-Nr eller D-Nr)"

Reply:
Ser sånn ut ja. Har du trua på erDnr implementasjonen?